### PR TITLE
V0.4

### PR DESCRIPTION
**Pull request by charlesk92**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
+## [0.4.23] - (December 4, 2020)
 ### Added
-Rule Added (3):
-- [CR-309] Enforce the use of VPC Endpoint Gateways for S3s in VPCs
-- [CR-495] IMDSv2 should be used and IMDSv1 should be disabled.
-- [CR-543] Athena Workgroup query results should be configured to be encrypted at rest
+Rule Added (6):
+Harden VPCs with the use of VPC Endpoint gateways for the following services:
+- [CR-309] S3
+- [CR-597] DynamoDB for VPCs
```
Comment (yi2020) [inline CHANGELOG.md:8]: Should be just "DynamoDB" - note that the whole VPC vs route table point exists for S3 too

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
+- [CR-598] DynamoDB for Route Tables
+
+- [CR-516] Ensure API Gateway caching is encrypted
+- [CR-593] Ensure no human IAM users defined
+- [CR-744] Ensure Redshift clusters being created are set to be encrypted at rest
 
 Resources supported for our Context Engine:
 - Auto-scaling groups and potential EC2 creations.
-- Support for ECS service as a potential LB target
+- NAT Gateway objects.
```
Comment (yi2020) [inline CHANGELOG.md:17]: This was added a long time ago

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,20 +1,25 @@
...
+Harden VPCs with the use of VPC Endpoint gateways for the following services:
+- [CR-309] S3
+- [CR-597] DynamoDB for VPCs
+- [CR-598] DynamoDB for Route Tables
+
+- [CR-516] Ensure API Gateway caching is encrypted
+- [CR-593] Ensure no human IAM users defined
+- [CR-744] Ensure Redshift clusters being created are set to be encrypted at rest
 
 Resources supported for our Context Engine:
```
Comment (yi2020) [inline CHANGELOG.md:15]: Maybe say "New resources"?

File: test/README.md [hunk 4]
```diff
@@ -55,18 +55,23 @@ If you'd like to be even more creative, you can apply some of the resources firs
...
+  26. [Cloudfront distribution should encrypt data in transit](https://github.com/indeni/cloudrail-demo/tree/master/test/aws/terraform/test_ensure_cloudfront_distribution_encrypt_in_transit_rule/not_encrypted)
+  27. [VPC Peers should have routes defined with least-access](https://github.com/indeni/cloudrail-demo/tree/master/test/aws/terraform/test_vpc_peering_least_access/routes_too_permissive_wider_than_VPC)
+  28. [API Gateway caching should be encrypted](https://github.com/indeni/cloudrail-demo/tree/master/test/aws/terraform/encrypted_rest_api)
+  29. [No IAM user should be defined](https://github.com/indeni/cloudrail-demo/tree/master/test/aws/terraform/iam_no_human_users_role)
+  30. [Redshift clusters should be encrypted at rest](https://github.com/indeni/cloudrail-demo/tree/master/test/aws/terraform/ensure_redshift_cluster_created_encrypted_rule)
+  31. [Enforce the use of VPC Endpoint gateways for S3](https://github.com/indeni/cloudrail-demo/tree/master/test/aws/terraform/s3_vpce_gateway_not_used_rule)
```
Comment (yi2020) [inline test/README.md:73]: Seems duplicate, see 25

File: CHANGELOG.md (hunk 3 above)
Comment (charlesk92) [reply CHANGELOG.md:15]: Redundant since it is all under the "added" section.

File: CHANGELOG.md [hunk 5]
```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 Rule Added (6):
 Harden VPCs with the use of VPC Endpoint gateways for the following services:
 - [CR-309] S3
-- [CR-597] DynamoDB for VPCs
+- [CR-597] DynamoDB
 - [CR-598] DynamoDB for Route Tables
```
Comment (yi2020) [inline CHANGELOG.md:9]: This line should also be removed.

File: test/README.md [hunk 6]
```diff
@@ -7,7 +7,7 @@ To try any of these test cases, go to the specific test case's directory, create
 ```
 ~/test/aws/terraform/public_access_security_groups_port_rule/port_22_allowed_from_internet_to_ec2_explicit # terraform init
 ~/test/aws/terraform/public_access_security_groups_port_rule/port_22_allowed_from_internet_to_ec2_explicit # terraform plan -out=plan.out
-~/test/aws/terraform/public_access_security_groups_port_rule/port_22_allowed_from_internet_to_ec2_explicit # cloudrail run
+~/test/aws/terraform/public_access_security_groups_port_rule/port_22_allowed_from_internet_to_ec2_explicit # cloudr`ail run
```
Comment (yi2020) [inline test/README.md:10]: Typo

File: test/README.md [hunk 7]
```diff
@@ -70,7 +70,6 @@ If you'd like to be even more creative, you can apply some of the resources firs
   28. [API Gateway caching should be encrypted](https://github.com/indeni/cloudrail-demo/tree/master/test/aws/terraform/encrypted_rest_api)
   29. [No IAM user should be defined](https://github.com/indeni/cloudrail-demo/tree/master/test/aws/terraform/iam_no_human_users_role)
   30. [Redshift clusters should be encrypted at rest](https://github.com/indeni/cloudrail-demo/tree/master/test/aws/terraform/ensure_redshift_cluster_created_encrypted_rule)
-  31. [Enforce the use of VPC Endpoint gateways for S3](https://github.com/indeni/cloudrail-demo/tree/master/test/aws/terraform/s3_vpce_gateway_not_used_rule)
   32. [Enforce the use of VPC Endpoint gateways for DynamoDB for VPCs](https://github.com/indeni/cloudrail-demo/tree/master/test/aws/terraform/dynamodb_vpce_gateway_not_used_rule)
```
Comment (yi2020) [inline test/README.md:73]: Should be 31 now

**Review by yi2020 — APPROVED** [review]

**Pull request merged**